recipe/meta.yaml

```
@@ -12,13 +12,9 @@ source:
   fn: conda-{{ version }}.tar.gz
   url: https://github.com/conda/conda/releases/download/{{ version }}/conda-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # Backport https://github.com/conda/conda/pull/14117 for changes in the
-    # Python 3.13+ logging module internals
-    - patches/0001-GH-14117-logging-lock-internal-changes-py313.patch
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv && {{ PYTHON }} -m conda init --install
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv && {{ PYTHON }} -I -m conda init --install
```

Avoid importing `conda/` from `$SRC_DIR`, which might result in [`CONDA_PACKAGE_SOURCE`](https://github.com/conda/conda/blob/89965c01801bd6511c0db5d0944f55108c16f3da/conda/__init__.py#L56) resolving to the wrong thing (it should be a location in `$PREFIX`, not `$SRC_DIR`).

Alternatively, bring in https://github.com/conda-forge/conda-feedstock/pull/204.

---

Ugh, looks like this messes with the cross-compilation setup in Python. Maybe using `python` directly works better?